iOS Client/HTTPClient/MBClient.swift

```diff
@@ -127,6 +127,23 @@ class MBClient: NSObject {
         }.resume()
     }
     
+    func getPodcastAudio(for podcast: MBPodcast, completion: @escaping (Data?, Error?) -> Void ) {
```

Comment:
It seems like the AVPlayer handles this for us down below. Is this function used?

Reply:
I had this before I realized I could just pass a url. I can remove